File: app/src/main/java/com/wiktor/weater1/newWeaher/weatherDetalisation/model/WeatherModelForView.java
package com.wiktor.weater1.newWeaher.weatherDetalisation.model;

public class WeatherModelForView {
    String date;
    Double maxTemperature;
    Double minTemperature;
    String imageUrl;

    public WeatherModelForView(String date, Double maxTemperature, Double minTemperature, String imageUrl) {
        this.date = date;
        this.maxTemperature = maxTemperature;
        this.minTemperature = minTemperature;
        this.imageUrl = imageUrl;
    }

    public String getDate() {
        return date;
    }

    public Double getMaxTemperature() {
        return maxTemperature;
    }

    public Double getMinTemperature() {
        return minTemperature;
    }

    public String getImageUrl() {
        return imageUrl;
    }
}

File: app/src/main/java/com/wiktor/weater1/newWeaher/weatherDetalisation/model/WeatherModelForView.kt
package com.wiktor.weater1.newWeaher.weatherDetalisation.model

class WeatherModelForView(date: String, maxTemperature: Double?, minTemperature: Double?, imageUrl: String) {
    var date: String
        internal set
    var maxTemperature: Double? = null
        internal set
    var minTemperature: Double? = null
        internal set
    var imageUrl: String
        internal set

    init {
        this.date = date
        this.maxTemperature = maxTemperature
        this.minTemperature = minTemperature
        this.imageUrl = imageUrl
    }
}